performance_regression_test.py

```diff
@@ -891,3 +891,65 @@ def test_latency_write_with_upgrade(self):
     def test_latency_mixed_with_upgrade(self):
         self._prepare_latency_with_upgrade()
         self.run_workload_and_upgrade(stress_cmd=self.params.get('stress_cmd_m'))
+
+
+class PerformanceRegressionMaterializedViewLatencyTest(PerformanceRegressionTest):
+
+    def test_read_mv_latency(self):
+        # next 3 lines, is a workaround to have it working inside `latency_calculator_decorator`
+        self.cluster = self.db_cluster  # pylint: disable=attribute-defined-outside-init
```

Comment:
I'm not sure if needed anymore, recently was fixed

Reply:
this hack still exists in PerformanceRegressionUpgradeTest.run_workload_and_upgrade where latency_calculator_decorator is used